tests/windows_test.go

```diff
@@ -255,6 +259,77 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 					`DNSDomain[\n\r\t ]+`+expectedSearchDomain+`[\n\r\t ]+`)
 			}, 360)
 		})
+
+		Context("VMI with bridge binding is created", func() {
+			var winVmiIp string
+			var winPodIp string
+
+			BeforeEach(func() {
+				windowsVMI.Spec.Domain.Devices.Interfaces = []v1.Interface{
+					{
+						Name: "default",
+						InterfaceBindingMethod: v1.InterfaceBindingMethod{
+							Bridge: &v1.InterfaceBridge{},
+						},
+					},
+				}
+
+				By("Starting Windows VirtualMachineInstance with bridge binding")
+				windowsVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(windowsVMI)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 360)
```

Comment:
Is this also waiting for the guest-agent to be up and running?
If it is not, then there is a need to add a wait for it as well, otherwise, the reported address in the VMI status may just be the IP seen on the pod.

The test itself seems raceful to me (even if you wait for the guest-agent here), especially if you do not make sure that the information you got is from the guest-agent.
Maybe when you add the reason of why this specific test was added, I will understand why we need this test (when we have the other one that checks connectivity).
It may even be worth merging the two.

Reply:
Relevant issues:
https://issues.redhat.com/browse/CNV-14301
https://github.com/kubevirt/kubevirt/issues/6448
https://github.com/kubevirt/kubevirt/issues/6462
This test supposed to cover the fix above

Reply:
AFAIK, there is no guest agent in this case. 
The ping test supposed to make sure that the vm's ip is the correct one.